.github/workflows/config.yml

```diff
@@ -22,13 +22,13 @@ jobs:
 
       - name: Run buildstockbatch
         run: |
-          pip install git+https://github.com/NREL/buildstockbatch.git@develop
+          pip install git+https://github.com/NREL/buildstockbatch.git@os330
```

Comment:
Change this back to `develop` after https://github.com/NREL/buildstockbatch/pull/251 gets merged.